package.json

```
@@ -40,16 +40,18 @@
     "clean": "npm-run-all -p clean:*",
     "clean:components-build": "rimraf packages/components/build",
     "clean:outputs": "rimraf -g output/**/src",
+    "clean:patternhub-pages": "rimraf -g showcases/patternhub/pages/components/*/*",
     "clean:patternhub-src-ts": "rimraf -g showcases/patternhub/components/src/**/*.ts",
     "clean:patternhub-src-tsx": "rimraf -g showcases/patternhub/components/src/**/*.tsx",
-    "clean:patternhub-pages": "rimraf -g showcases/patternhub/pages/components/*/*",
     "dev": "node scripts/dev.js",
     "dev:angular-components": "npm run dev:angular --workspace=@db-ui/components",
     "dev:plain-html": "npm run dev:html --workspace=@db-ui/components",
     "dev:react-components": "npm run dev:react --workspace=@db-ui/components",
     "dev:sass": "npm run dev:scss --workspace=@db-ui/components",
     "dev:vue-components": "npm run dev:vue --workspace=@db-ui/components",
     "generate:component": "npm run generate:component --workspace=@db-ui/components",
+    "generate:fonts": "npm run generate:fonts --workspace=@db-ui/foundations",
```

```suggestion
    "generate:fonts": "npm run generate:icon-fonts --workspace=@db-ui/foundations",
```

What about renaming this for "clarity" ?